raiden/settings.py

```diff
@@ -6,7 +6,7 @@
 DEFAULT_REVEAL_TIMEOUT = 30
 DEFAULT_SETTLE_TIMEOUT = DEFAULT_REVEAL_TIMEOUT * 20
 DEFAULT_EVENTS_POLL_TIMEOUT = 0.5
-DEFAULT_POLL_TIMEOUT = 60
+DEFAULT_POLL_TIMEOUT = 120
```

Comment:
Tested on a slow machine, 60 seconds is not enough (this was required additionally to the DAG memory check)